content/chapters/compute/lab/support/user-level-threads/simple.cc

```diff
@@ -0,0 +1,25 @@
+#include <boost/fiber/all.hpp>
+#include <vector>
+
+#include <stdio.h>
+
+using namespace std;
+
+void hello(void)
+{
+    printf("Hello World\n");
+}
+
+int main(void)
+{
+    int nr_fibers = 5;
```

Comment:
```suggestion
    int num_fibers = 5;
```
`nr` is specific to Romanian. You may alternatively use `no_fibers`, but I think I've seen `num_` more often.

And why don't you turn this into a `#define` and then tell students to modify it?